usm_rest_api/v1/views/tasks.py

```diff
@@ -0,0 +1,328 @@
+import logging
+
+from celery import shared_task
+from celery import states
+from celery import current_task
+
+from usm_rest_api.models import Cluster
+from usm_rest_api.models import Host
+
+from usm_rest_api.v1.views import utils as usm_rest_utils
+
+
+log = logging.getLogger('django.request')
+
+
+@shared_task
+def createCephCluster(cluster_data):
+    log.debug("Inside createCephCluster Async Task")
+    log.debug(cluster_data)
+    nodelist = []
+    failedNodes = []
+    noOfNodes = 0
+    if 'nodes' in cluster_data:
+        nodelist = cluster_data['nodes']
+        noOfNodes = len(nodelist)
+        del cluster_data['nodes']
+
+    # create the cluster
+    try:
+        current_task.update_state(state='CREATE_CLUSTER')
```

Comment:
What I meant was updating Db table. I guess this will create confusion, so I will remove this custom state